src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/RedditAnalyzer.scala

```diff
@@ -0,0 +1,22 @@
+package com.microsoft.partnercatalyst.fortis.spark.analyzer
+
+import java.time.Instant.now
+import java.util.UUID.randomUUID
+
+import com.github.catalystcode.fortis.spark.streaming.reddit.dto.RedditObject
+import com.microsoft.partnercatalyst.fortis.spark.transforms.image.ImageAnalyzer
+
+@SerialVersionUID(100L)
+class RedditAnalyzer extends Analyzer[RedditObject] with Serializable with AnalysisDefaults.EnableAll[RedditObject] {

```

Comment:
nit: add newline and indent before `with AnalysisDefaults.EnableAll[RedditObject]`